### PR TITLE
Patched issue #263 (if needed)

### PR DESCRIPTION
**Pull request by ikarius**
ASIFormDataRequest has a default 'GET' method (should be 'POST' by
default)

### CODE REVIEW

File: Classes/ASIFormDataRequest.m [hunk 1]
```diff
@@ -49,6 +49,7 @@
 	self = [super initWithURL:newURL];
 	[self setPostFormat:ASIURLEncodedPostFormat];
 	[self setStringEncoding:NSUTF8StringEncoding];
+    [self setRequestMethod:@"POST"];
```
Comment (greenisus) [inline Classes/ASIFormDataRequest.m]: this looks great all around, but can you make the whitespace here consistent with the existing code?

Comment (ikarius) [inline Classes/ASIFormDataRequest.m]: Sorry...

Correct now

Le 7 octobre 2011 23:53, Michael Mayo <
reply@reply.github.com> a
écrit :

> > @@ -49,6 +49,7 @@
> >       self = [super initWithURL:newURL];
> >       [self setPostFormat:ASIURLEncodedPostFormat];
> >       [self setStringEncoding:NSUTF8StringEncoding];
> > -    [self setRequestMethod:@"POST"];
> 
> this looks great all around, but can you make the whitespace here
> consistent with the existing code?
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/pokeb/asi-http-request/pull/268/files#r160536

## 

Frédéric VERGEZ --
Développeur iOs / Android / Scala
Tél. : +33 981 847 699  - @: fred@ikarius.com

**Comment by greenisus** [comment]
thanks again!

**Comment by ikarius** [comment]
De nada !

**Pull request merged**